<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MineralogyDataForm</class>
 <widget class="QWidget" name="MineralogyDataForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>206</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
   <property name="horizontalSpacing">
    <number>30</number>
   </property>
   <property name="verticalSpacing">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>9</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="md_formation_types_label">
       <property name="text">
        <string notr="true">Formation Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="md_formation_types_combobox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="md_clays_types_label">
       <property name="text">
        <string notr="true">Clay Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="md_clays_types_combobox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="md_clay_content_label">
       <property name="text">
        <string notr="true">Clay Content</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="md_cementing_material_carbonate_label">
       <property name="text">
        <string notr="true">Is the cementing material carbonate?</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="md_cementing_material_carbonate_checkbox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="md_clay_content_lineedit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="horizontalSpacing">
      <number>9</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="pd_formation_damage_label">
       <property name="text">
        <string notr="true">Formation damage</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTableView" name="pd_formation_damage_tableview">
       <property name="styleSheet">
        <string notr="true">selection-background-color: rgb(158, 155, 255);</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
